domain/src/main/java/edu/nextstep/camp/calculator/domain/CalculateOrder.kt

```diff
@@ -0,0 +1,15 @@
+package edu.nextstep.camp.calculator.domain
+
+enum class CalculateOrder {
+    NUMBER_FIRST,
+    NUMBER_SECOND,
+    OPERATION;
+
+    fun getNextCalculateOrder(): CalculateOrder {
+        return when (this) {
+            NUMBER_FIRST -> OPERATION
+            OPERATION -> NUMBER_SECOND
+            NUMBER_SECOND -> OPERATION
```

Comment:
```suggestion
            NUMBER_FIRST,
            NUMBER_SECOND -> OPERATION
            OPERATION -> NUMBER_SECOND
```
같은 값을 리턴하는 경우에는 묶어서 처리할 수 있어요

Reply:
d2cb160